stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/filepreview/MediaAttachmentPreviewHandler.kt

```diff
@@ -16,20 +16,18 @@ public class MediaAttachmentPreviewHandler(private val context: Context) : Attac
         val type = attachment.type
 
         if (assetUrl.isNullOrEmpty()) return false
-        if (mimeType.isNullOrEmpty()) return false
+
+        // Check if the base of Attachment.mimeType is audio or video
+        if (!mimeType.isNullOrEmpty() && (mimeType.contains(ModelType.attach_audio) || mimeType.contains(ModelType.attach_video))) return true
```

Comment:
This in certain cases causes false positives such as in the case of `.avi` files which can be opened by us but not played by ExoPlayer. 

This should still be better than causing false negatives by unwittingly restricting mime types.